api/index.js

```diff
@@ -17,17 +17,9 @@ api.use(jwt({ secret: process.env.JWT_SECRET }).unless((request) => {
 }))
 
 api.use(function (request, response, next) {
-  if (process.env.ENVIRONMENT !== 'production') {
-    let incomingOrigin = request.headers['Origin']
-
-    if (incomingOrigin && (
-        incomingOrigin.startsWith('http://content-scoring-local.cru.org:') ||
-        incomingOrigin === 'https://content-scoring-stage.cru.org')) {
-      response.header('Access-Control-Allow-Origin', incomingOrigin)
-      response.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept')
-    }
-  }
-
+  response.header('Access-Control-Allow-Origin', '*')
```

Comment:
Will this open a security vulnerability to DoS attacks?

Reply:
No, as any malicious request can ignore CORS and hit the API with or without this. If necessary, we could whitelist which domains we will allow, but I don't think it's necessary.